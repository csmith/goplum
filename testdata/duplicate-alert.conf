alert debug.sysout "alert1" {}
alert http.webhook "alert1" {
    url = "https://example.com/"
}
