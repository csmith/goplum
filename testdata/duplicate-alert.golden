"alert defined multiple times: alert1"